### PR TITLE
Add post about new Emacs integration package.

### PR DESCRIPTION
**Pull request by Mike-Woolley**

### CODE REVIEW

File: _posts/2022-12-16-emacs-package.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Emacs Package"
+date: 2022-12-16 18:10:00 +0000
+---
+
+Boxes integration is now available as an Emacs package 😊<br/>
+[![MELPA](https://melpa.org/packages/boxes-badge.svg)]({{ site.baseurl }}/editors-emacs.html)
```
Comment (tsjensen) [inline _posts/2022-12-16-emacs-package.md:8]: I was first confused that the MELPA link didn't bring me to MELPA, so I would suggest that we have two links: one to MELPA and another one to our own page.
```suggestion
[Boxes integration for Emacs]({{ site.baseurl }}/editors-emacs.html) is now available as an Emacs package!<br/>
[![MELPA](https://melpa.org/packages/boxes-badge.svg)](https://melpa.org/#/boxes)
```

Comment (Mike-Woolley) [reply _posts/2022-12-16-emacs-package.md:8]: Damn 100% failure rate 🤢

If it's confusing then of course I'll change it.

Thanks,
Mike

Comment (tsjensen) [reply _posts/2022-12-16-emacs-package.md]: > Damn 100% failure rate

What I see is that you have a 100% *success* rate!

Comment (Mike-Woolley) [reply _posts/2022-12-16-emacs-package.md]: Looks good Thomas,

Thanks,
Mike

**Pull request merged**